### DISCUSSION TITLE
Concurrent Controller and Hand Tracking

### DISCUSSION THREAD

**Discussion by szigetlaki** [alvr-org/ALVR#2464] [General]
Hi Guys, the latest Meta SDK allows the use of both controllers and hand tracking simultaneously. My question is, could someone help implement this feature in ALVR? We can also pay in exchange for the help.

Thanks, Zsolt

**Comment by zmerp** [comment]
Hi Zsolt,
Yes, It is possible. SteamVR does not support concurrent controllers and hand trackers, so we would have to map them to body trackers, but  we don't have a remapping infrastructure for trackers. I'd say I could help with this as an ad-hoc solution. Please contact me by email (visible in my account).

Riccardo
